src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/RcaController.java

```diff
@@ -85,6 +88,11 @@
   // This needs to be volatile as the RcaConfPoller writes it but the Nanny reads it.
   private static volatile boolean rcaEnabled = false;
 
+  private String mutedRcasDefaultValue = "";
```

Comment:
Maybe `final` ?

Reply:
Agreed, makes more sense to have it as `final`